subsys/net/lib/http/http_client.c

```diff
@@ -208,7 +209,7 @@ static int on_header_field(struct http_parser *parser, const char *at,
 	return 0;
 }
 
-#define MAX_NUM_DIGITS	16
+#define MAX_NUM_DIGITS  16
```

Comment:
Same comment to this one. If you want to fix these white space issues, then you can introduce a separate commit that does that (could be part of this PR of course)

Reply:
ditto